internal/cltest/cltest.go

```diff
@@ -276,7 +276,7 @@ func ObserveLogs() *observer.ObservedLogs {
 	return observed
 }
 
-func FixtureCreateJobViaWeb(t *testing.T, app *TestApplication, path string) models.JobSpec {
+func FixtureCreateJobViaWeb(t assert.TestingT, app *TestApplication, path string) models.JobSpec {
```

Comment:
Hey there, what's the reason for changing this to `assert.TestingT`? If there is no strong reason, can we change it back to `*testing.T` for consistency with the rest of the test suite and `*testing.B`? If so, this file doesn't need to change at all.